pyseir/inference/model_fitter.py

```diff
@@ -366,9 +393,17 @@ def run_model(self, R0, eps, t_break, log10_I_initial):
         model: SEIRModel
             The SEIR model that has been run.
         """
-        suppression_policy = suppression_policies.generate_two_step_policy(
-            self.t_list, eps, t_break
-        )
+        if self.suppression_policy is SuppressionPolicy.TWO_STAGE:
+            suppression_policy = suppression_policies.generate_two_step_policy(
+                self.t_list, eps, t_break
+            )
+        elif self.suppression_policy is SuppressionPolicy.RT:
+            suppression_policy = suppression_policies.generate_rt_inference_policy(
+                t0=self.ref_date + timedelta(days=t0),
+                fips=self.fips,
+                future_suppression='current',
+                # R0=R0 # TODO: TBD if this should be fit...
```

Comment:
Here we need to probably instead make this a scalar which makes`R_seir(t) = 1+ alpha * (R(t) - 1)`